src/react/hooks/paragon/useParagonTheme.test.js

```diff
@@ -0,0 +1,177 @@
+import { act, renderHook } from '@testing-library/react';
+
+import useParagonTheme from './useParagonTheme';
+import { getConfig } from '../../../config';
+import { logError } from '../../../logging';
+
+jest.mock('../../../logging');
+
+const PARAGON_THEME_URLS = {
+  core: {
+    urls: {
+      default: 'core.css',
+    },
+  },
+  defaults: {
+    light: 'light',
+    dark: 'dark',
+  },
+  variants: {
+    light: {
+      urls: {
+        default: 'https://cdn.jsdelivr.net/npm/@edx/paragon@21.0.0/dist/light.min.css',
+      },
+    },
+    dark: {
+      urls: {
+        default: 'https://cdn.jsdelivr.net/npm/@edx/paragon@21.0.0/dist/dark.min.css',
```

Comment:
[non-blocking nit] Should these CDN urls point to `@openedx/paragon@23` at this point? These files technically don't exist on v21.